src/styles/components/alert.less

```diff
@@ -1,7 +1,6 @@
 // Quick and dirty bootstrap-style "alert" box from old website.
 // This supports markup in already-existing docs. We may want to change it.
 .alert {
-  border: 1px solid #f3f3f3;
```

Comment:
Change from 1px to 2px for added visibility since we're eliminating the box-shadow

Reply:
Fixed with 5dbf8c7